packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/quoteFeatures.ts

```diff
@@ -74,16 +76,36 @@ function splitQuote(event: PluginKeyboardEvent, editor: IEditor) {
     editor.addUndoSnapshot(() => {
         let childOfQuote = cacheGetQuoteChild(event, editor);
         let parent: Node;
+        let shouldClearFormat: boolean;
         if (getTagOfNode(childOfQuote) == QUOTE_TAG) {
             childOfQuote = wrap(toArray(childOfQuote.childNodes));
         }
         parent = splitBalancedNodeRange(childOfQuote);
+        shouldClearFormat = isStyledBlockquote(parent);
         unwrap(parent);
         editor.select(childOfQuote, PositionType.Begin);
+
+        if (shouldClearFormat) {
+            clearFormat(editor);
+        }
     });
     event.rawEvent.preventDefault();
 }
 
+const isStyledBlockquote = (element: Node) => {
+    if (
+        element &&
+        safeInstanceOf(element, 'HTMLQuoteElement') &&
+        element.style.borderLeft &&
+        element.style.borderColor &&
+        element.style.paddingLeft &&
+        element.style.color
+    ) {
+        return true;
+    }
+    return false;
+};
+
 /**
```

Comment:
Added this check, so we verify that the QuoteElement is styled and it is not a blockquote used in indentation